octoprint_mrbeam/gcodegenerator/jobtimeestimation.py

```diff
@@ -67,7 +67,7 @@ def on_event(self, event, payload):
             estimation_thread.daemon = True
             estimation_thread.start()
 
-        if event == OctoPrintEvents.CLIENT_OPENED:
+        if event == OctoPrintEvents.CLIENT_OPENED and self._last_estimation != -1:
```

Comment:
What problem does this solve or what is the benefit?

Reply:
Fewer events. This one is entirely unnecessary if no estimations had been made before-hand.

If an estimation is being calculated, then we want to notify front end, but let's check if there is one running first.

Reply:
TODO : Check if a "ready to laser state" is not communicated properly when reloading the page

Reply:
ok, fine with me

Reply:
Develop branch did not display the ready to laser page when reloading prior to the merge. Therefore it is not blocking.